A bunch of tests fail when sqlite is the only db.

diff --git a/tests/db/have-db.in b/tests/db/have-db.in
index 917ceb4..96afca9 100644
--- a/tests/db/have-db.in
+++ b/tests/db/have-db.in
@@ -64,7 +64,6 @@ while [ $# != 0 ] ; do
     db4:*) exit 0 ;;
     db3:*) exit 0 ;;
     lmdb:*) exit 0 ;;
-    sqlite:*) exit 0 ;;
     esac
     shift
 done
